### ISSUE TITLE
[BUG] Video upload not working

### ISSUE THREAD

**Issue by spctra** [ohld/igbot#1267]
**Describe the bug**

Been trying out the bot, looks like everything's working for me except the upload video function.
Tried both with my own media and with the provided example.
The bot returns a successful response yet no video is ever uploaded.

**Log**

```
2020-01-27 23:03:25,356 - INFO - Instabot Started
2020-01-27 23:03:28,570 - INFO - Recovery from test_my_shit_yes_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 105.0.0.18.119 Android (28/9.0; 420dpi; 1080x2260; OnePlus; GM1903; OnePlus7; qcom; en_US)
- phone_id=f1f7cdc4-639f-4610-be33-c1d109aee8a4
- uuid=19cc0afb-9b30-4a31-9255-6fe3fd4b916a
- client_session_id=531fa12d-e2f9-4936-b44b-3af79214022d
- device_id=android-6350d9791eaee882
2020-01-27 23:03:28,571 - INFO - LOGIN FLOW! Just logged-in: False
2020-01-27 23:03:33,407 - INFO - Logged-in successfully as 'test_my_shit_yes'!
2020-01-27 23:03:33,430 - INFO - Checking 02.mov
2020-01-27 23:03:33,430 - INFO - Uploading video `02.mov` with caption: `I'm the caption of #horizontal #mov #video that will be resized and cropped but not cut, with a mention to @maxdevblock`
2020-01-27 23:03:36,144 - INFO - Started uploading '\instabot\examples\video/media/02.mov'
2020-01-27 23:04:07,477 - INFO - Video '\instabot\examples\video/media/02.mov' uploaded
2020-01-27 23:04:07,479 - INFO - Succesfully uploaded: 02.mov
2020-01-27 23:04:07,481 - INFO - Total requests: 25
```

**Additional context**

Running with Python 3.7.4 on Windows 10.

**Comment by bruvv** [comment]
You are probably blocked. Try with a different account that never did anything with a bot

**Comment by spctra** [comment]
@bruvv I just tried opening a new account and running the bot.
Got the same result.
Just to clarify, everything else works(as far as I tested), only posting videos doesn't.

```2020-01-29 17:36:56,168 - INFO - PRE-LOGIN FLOW!...
2020-01-29 17:36:56,590 - DEBUG - <Response [200]>
2020-01-29 17:36:56,808 - DEBUG - <Response [200]>
2020-01-29 17:36:57,047 - DEBUG - <Response [200]>
2020-01-29 17:36:57,265 - DEBUG - <Response [200]>
2020-01-29 17:36:57,488 - DEBUG - <Response [200]>
2020-01-29 17:36:58,590 - DEBUG - <Response [200]>
2020-01-29 17:36:58,591 - INFO - Logged-in successfully as 'this_is_my_test_yes'!
2020-01-29 17:36:58,592 - INFO - LOGIN FLOW! Just logged-in: True
2020-01-29 17:36:58,881 - DEBUG - <Response [200]>
2020-01-29 17:36:59,452 - DEBUG - <Response [200]>
2020-01-29 17:37:02,675 - DEBUG - <Response [200]>
2020-01-29 17:37:03,005 - DEBUG - <Response [200]>
2020-01-29 17:37:03,302 - DEBUG - <Response [200]>
2020-01-29 17:37:03,585 - DEBUG - <Response [200]>
2020-01-29 17:37:03,871 - DEBUG - <Response [200]>
2020-01-29 17:37:04,205 - DEBUG - <Response [200]>
2020-01-29 17:37:04,477 - DEBUG - <Response [200]>
2020-01-29 17:37:04,796 - DEBUG - <Response [200]>
2020-01-29 17:37:05,592 - DEBUG - <Response [200]>
2020-01-29 17:37:05,907 - DEBUG - <Response [200]>
2020-01-29 17:37:06,172 - DEBUG - <Response [200]>
2020-01-29 17:37:06,532 - DEBUG - <Response [200]>
2020-01-29 17:37:08,142 - DEBUG - <Response [200]>
2020-01-29 17:37:08,153 - INFO - Checking 02.mov
2020-01-29 17:37:08,154 - INFO - Uploading video `02.mov` with caption: `I'm the caption of #horizontal #mov #video that will be resized and cropped but not cut, with a mention to @maxdevblock`
2020-01-29 17:37:09,186 - INFO - Started uploading '\instabot\examples\video/media/02.mov'
Analizing `\instabot\examples\video/media/02.mov`
FOUND w:1440, h:720, rotation=0, ratio=2.0
Horizontal video
Cropping video
Resizing video
2020-01-29 17:37:38,649 - DEBUG - <Response [200]>
2020-01-29 17:37:38,902 - DEBUG - <Response [200]>
2020-01-29 17:37:38,907 - INFO - Video '\instabot\examples\video/media/02.mov' uploaded
2020-01-29 17:37:38,914 - INFO - Succesfully uploaded: 02.mov
2020-01-29 17:37:38,916 - INFO - Total requests: 23```

**Comment by quasa0** [comment]
> **Describe the bug**
> 
> Been trying out the bot, looks like everything's working for me except the upload video function.
> Tried both with my own media and with the provided example.
> The bot returns a successful response yet no video is ever uploaded.
> 
> **Log**
> 
> ```
> 2020-01-27 23:03:25,356 - INFO - Instabot Started
> 2020-01-27 23:03:28,570 - INFO - Recovery from test_my_shit_yes_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
> - user-agent=Instagram 105.0.0.18.119 Android (28/9.0; 420dpi; 1080x2260; OnePlus; GM1903; OnePlus7; qcom; en_US)
> - phone_id=f1f7cdc4-639f-4610-be33-c1d109aee8a4
> - uuid=19cc0afb-9b30-4a31-9255-6fe3fd4b916a
> - client_session_id=531fa12d-e2f9-4936-b44b-3af79214022d
> - device_id=android-6350d9791eaee882
> 2020-01-27 23:03:28,571 - INFO - LOGIN FLOW! Just logged-in: False
> 2020-01-27 23:03:33,407 - INFO - Logged-in successfully as 'test_my_shit_yes'!
> 2020-01-27 23:03:33,430 - INFO - Checking 02.mov
> 2020-01-27 23:03:33,430 - INFO - Uploading video `02.mov` with caption: `I'm the caption of #horizontal #mov #video that will be resized and cropped but not cut, with a mention to @maxdevblock`
> 2020-01-27 23:03:36,144 - INFO - Started uploading '\instabot\examples\video/media/02.mov'
> 2020-01-27 23:04:07,477 - INFO - Video '\instabot\examples\video/media/02.mov' uploaded
> 2020-01-27 23:04:07,479 - INFO - Succesfully uploaded: 02.mov
> 2020-01-27 23:04:07,481 - INFO - Total requests: 25
> ```
> 
> **Additional context**
> 
> Running with Python 3.7.4 on Windows 10.

Could you show the full code you're running so that I could reproduce it on my account?

**Comment by gmyrzamuratov** [comment]
Yes, I have the same issues. Successfully uploaded, but the video didn't appear. I tried to upload this video from my phone's Instagram app and post appeared.  I didn't get the error that I blocked.

> 2020-01-31 17:35:01,183 - INFO - Started uploading 'C:\Users\Admin\Desktop\python\sazalem\next\media\Joldasbek.mp4'
Analizing `C:\Users\Admin\Desktop\python\sazalem\next\media\Joldasbek.mp4`
FOUND w:720, h:720, rotation=0, ratio=1.0
Square video
2020-01-31 17:35:27,028 - DEBUG - POST to endpoint: media/configure/?video=1 returned response: <Response [200]>
2020-01-31 17:35:27,287 - DEBUG - POST to endpoint: qe/expose/ returned response: <Response [200]>
2020-01-31 17:35:27,291 - INFO - Video 'C:\Users\Admin\Desktop\python\sazalem\next\media\Joldasbek.mp4' uploaded
2020-01-31 17:35:27,294 - INFO - Succesfully uploaded: Joldasbek.mp4
2020-01-31 17:35:27,299 - INFO - Total requests: 27

>

**Comment by fargolk** [comment]
same here `INFO - Video .. uploaded`   getting no error but nothing there

**Comment by lsilva5455** [comment]
I have same issues, any news??
* instabot 0.108.0 and 0.109.0
* Python 3.7.0  and Python 3.6.8
* Raspbian GNU/Linux 9 (stretch) 

2020-02-22 10:43:03,833 - INFO - Started uploading '/home/instabot/examples/video/media/60s9.mp4'
Analizing '/home/instabot/examples/video/media/60s9.mp4'
FOUND w:720, h:1280, rotation=0, ratio=0.5625
Vertical video
Cropping video
2020-02-22 10:43:41,060 - DEBUG - POST to endpoint: media/configure/?video=1 returned response: <Response [200]>
2020-02-22 10:43:44,988 - DEBUG - POST to endpoint: qe/expose/ returned response: <Response [200]>
2020-02-22 10:26:22,734 - INFO - Succesfully uploaded: 60s8.mp4

**Comment by majda107** [comment]
same issue here

**Comment by ufilme** [comment]
same

**Comment by vladislovenx** [comment]
Have the same problems with upload photo and video to stories
